cocotb/_build_libs.py

```diff
@@ -38,6 +38,28 @@ def run(self):
 
         super().run()
 
+    # On osx need extra extra rpath with install_name_tool to find libraries
+    if sys.platform == "darwin":
+        def build_extension(self, ext):
+            super().build_extension(ext)
+
+            subprocess.run(
+                [
+                    "install_name_tool",
+                    "-add_rpath",
+                    "@loader_path",
+                    os.path.join(self.build_lib, ext._file_name),
+                ]
+            )
+            subprocess.run(
+                [
+                    "install_name_tool",
+                    "-add_rpath",
+                    "@loader_path/libs",
+                    os.path.join(self.build_lib, ext._file_name),
+                ]
+            )
```

Comment:
This is the osx version of ` extra_link_args=["-Wl,-rpath,$ORIGIN/libs"],`, right?

Reply:
Yes but not exactly. On osx this has to be set on a library that is being loaded and with '-rpath' on a library that loads. Not sure what is all the logic here.